dist/linux/byedpi_system.service

```diff
@@ -1,14 +1,17 @@
 [Unit]
-Description=byedpi
+Description=ByeDPI
 Documentation=https://github.com/hufrea/byedpi
 
 [Service]
-ExecStart=ciadpi --split 1 --disorder 3+s --mod-http=h,d --auto=torst --tlsrec 1+s
+StandardOutput=null
+StandardError=journal
+EnvironmentFile=-/etc/byedpi
+ExecStart=ciadpi $BYEDPI_OPTIONS
 TimeoutStopSec=5s
 LimitNOFILE=1048576
```

Comment:
@trevarj why exactly this number? I think this should be removed, like the limit written below.

Reply:
It can be removed. I took it from another service unit.

Reply:
@hufrea Should I remove
```service
TimeoutStopSec=5s
LimitNOFILE=1048576
```
or
```service
LimitNOFILE=1048576
LimitNPROC=512
```
?

As far as I understand, `TimeoutStopSec` will help if the process hangs. I've never had this problem with `ciadpi`, but I have had it with other processes/services in the past. I don't know how things are now, but in the past I've had problems where it would take 15 minutes to shut down the computer, due to a hung process and too long default timeouts.

Reply:
Remove these:

```
LimitNOFILE=1048576
LimitNPROC=512
```

I'm not actually sure what the effects will be long-term, but it should still work. These fields are for large/heavy services like docker or something.